packages/lesswrong/components/sunshineDashboard/RejectedReason.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { registerComponent, Components } from '../../lib/vulcan-lib';
+import Card from '@material-ui/core/Card';
+import { htmlToText } from 'html-to-text';
+
+const styles = (theme: ThemeType): JssStyles => ({
+  root: {
+    marginBottom: 4
+  },
+  shortReason: {
+    maxWidth: 500,
+    overflow: "hidden"
+  },
+  reasonTooltip: {
+    paddingTop: 4,
+    paddingBottom: 2,
+    paddingLeft: 12,
+    paddingRight: 16,
+    width: 400,
+    fontSize: '1rem',
+    marginBottom: 12
+  },
+});
+
+export const RejectedReason = ({classes, reason}: {
```

Comment:
I feel weird about this component in some way. I think either it should be titled something more specific like `RejectedReasonText` or I dunno. Possibly the logic for extracting the reason should be separated as a util function or something.